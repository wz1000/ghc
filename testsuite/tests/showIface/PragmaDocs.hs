module PragmaDocs where

{-# DEPRECATED contains "Use `elem` instead." #-}
contains :: (Eq a, Foldable f) => f a -> a -> Bool
contains = flip elem

{-# warning x, y "These are useless" #-}
x = ()
y = ()
