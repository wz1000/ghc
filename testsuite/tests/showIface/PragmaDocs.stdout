Warnings: x: "These are useless"
          y: "These are useless"
          contains: "Use `elem` instead."
trusted: safe
require own pkg trusted: False
docs:
  Just module header:
         Nothing
       declaration docs:
       arg docs:
       documentation structure:
         avails:
           [contains]
         avails:
           [x]
         avails:
           [y]
       named chunks:
       haddock options:
       language:
         Nothing
       language extensions:
         MonomorphismRestriction
         RelaxedPolyRec
         ForeignFunctionInterface
         ImplicitPrelude
         DoAndIfThenElse
         EmptyDataDecls
         PatternGuards
         NondecreasingIndentation
         TraditionalRecordSyntax
         MonadFailDesugaring
         StarIsType
